src/hooks/safe-apps/useRemoteSafeApps.ts

```diff
@@ -41,10 +41,18 @@ const useRemoteSafeApps = (tag?: SafeAppsTag): AsyncResult<SafeAppsResponse> =>
 
   const apps = useMemo(() => {
     if (!remoteApps || !tag) return remoteApps
-    return remoteApps.filter((app) => app.tags.includes(tag))
+    return remoteApps
+      .filter((app) => app.tags.includes(tag))
+      .sort((a, b) => {
+        return a.name < b.name ? -1 : a.name > b.name ? 1 : 0
+      })
```

Comment:
Can we avoid code repetition here?
```
const apps = tag ? apps.filter(tag) : apps
return apps.sort()
```

Reply:
Good catch! I created `sortedApps` to remove this but forgot to do so 🙈